cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -2203,6 +2206,53 @@ if(ARROW_WITH_RAPIDJSON)
   endif()
 endif()
 
+
+macro(build_qpl)
+  message(STATUS "Building QPL from source")
+  set(QPL_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/qpl_ep/src/qpl_ep-install")
+  if(MSVC)
+      set(QPL_STATIC_LIB_NAME qplstatic.lib)
+  else()
+    set(QPL_STATIC_LIB_NAME libqpl.a)
+  endif()
+  set(QPL_STATIC_LIB "${QPL_PREFIX}/lib64/${QPL_STATIC_LIB_NAME}")
```

Comment:
`lib64` is environment dependent. We can always use `lib` with `-DCMAKE_INSTALL_LIBDIR=lib`.